### PR TITLE
Mixin language fix.

### PR DESCRIPTION
**Pull request by damacus**
### Description

As per usual I went down a bit of a rabbit hole...

### Issues Resolved
- Fix dokken config 
- Remove make commands that are no longer understood by php make (and makes it fail)
- Remove matchers 

### Check List

- [ ] All tests pass. See <https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/TESTING.MD>
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable
- [ ] All commits have been signed for the Developer Certificate of Origin. See <https://github.com/chef-cookbooks/community_cookbook_documentation/blob/master/CONTRIBUTING.MD>

### CODE REVIEW

**Comment by tas50** [comment]
@damacus DCO

**Comment by tas50** [comment]
Also you turned on testing for opensuse that doesn't work

**Comment by damacus** [comment]
Yeah @tas50 https://github.com/coderanger/dco/issues/2

**Comment by damacus** [comment]
Weird that amazon is failing in Travis, as that's working locally.

File: resources/pear.rb [hunk 1]
```diff
@@ -18,21 +18,80 @@
...
+  end
+end
+
+action :install do
+  # If we specified a version, and it's not the current version, move to the specified version
+  install_version = new_resource.version unless new_resource.version.nil? || new_resource.version == current_resource.version
+
+  # If it's not installed at all or an upgrade, install it
+  if install_version || new_resource.version.nil?
+    description = "install package #{new_resource.package_name} #{install_version}"
```
Comment (tas50) [inline resources/pear.rb]: just collapse this down and avoid setting the variable

Comment (damacus) [reply resources/pear.rb]: Copy pasta, was working on getting the current value fixed :)

**Pull request merged**